test/integration/utility.cc

```diff
@@ -189,6 +190,7 @@ IntegrationUtil::makeSingleRequest(const Network::Address::InstanceConstSharedPt
                 Filesystem::fileSystemForTest(), random_generator);
   Event::DispatcherPtr dispatcher(api.allocateDispatcher("test_thread"));
   TestConnectionCallbacks connection_callbacks(*dispatcher);
+
```

Comment:
given the existing usage model and implementation of this, where mock_stats_store is instantiated a few lines up, I think it would be more consistent for you to instantiate a new `Quic::QuicStatNames` here, using `mock_stats_store.symbolTable()`.

That will avoid the new args to makeSingleRequest, and remove a large number of other changes in this PR I think.

Reply:
That's an great advice!
done.